src/fairchem/core/scripts/download_large_files.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+from urllib.request import urlretrieve
+
+FAIRCHEM_ROOT = Path(__file__).parents[4]
```

Comment:
Any chance we can re-use a function like, 
_get_project_root() 
or 
fairchem_root() ?